doc/api/index.md

```diff
@@ -46,6 +46,7 @@
 * [Performance hooks](perf_hooks.md)
 * [Policies](policy.md)
 * [Process](process.md)
+* [PromiseHooks](promise_hooks.md)
```

Comment:
If `PromiseHooks` is its own documentation entry, it should also be its own module.